<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="sizeIncrement">
   <size>
    <width>8</width>
    <height>0</height>
   </size>
  </property>
  <property name="font">
   <font>
    <pointsize>16</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(184, 255, 195);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QProgressBar" name="progressBar">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>270</y>
      <width>331</width>
      <height>23</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Broadway</family>
     </font>
    </property>
    <property name="value">
     <number>0</number>
    </property>
   </widget>
   <widget class="QLabel" name="PLANTA">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>20</y>
      <width>251</width>
      <height>81</height>
     </rect>
    </property>
    <property name="baseSize">
     <size>
      <width>0</width>
      <height>0</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>Broadway</family>
      <pointsize>20</pointsize>
      <underline>true</underline>
     </font>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 85, 0);

</string>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Plain</enum>
    </property>
    <property name="text">
     <string>PLANTA CLIKER:</string>
    </property>
    <property name="scaledContents">
     <bool>false</bool>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="margin">
     <number>2</number>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>120</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>SimSun-ExtB</family>
      <pointsize>14</pointsize>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="focusPolicy">
     <enum>Qt::ClickFocus</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 0, 127);
background-color: rgb(161, 246, 255);
background-color: rgb(120, 224, 255);
border-radius: 20px;
border-width: 1px;
border-style: solid;</string>
    </property>
    <property name="text">
     <string>REGAR</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>330</y>
      <width>141</width>
      <height>20</height>
     </rect>
    </property>
    <property name="sizeIncrement">
     <size>
      <width>9</width>
      <height>0</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>Candara Light</family>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">
color: rgb(85, 0, 0);</string>
    </property>
    <property name="lineWidth">
     <number>0</number>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>190</y>
      <width>449</width>
      <height>45</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Broadway</family>
      <pointsize>13</pointsize>
     </font>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="font">
        <font>
         <family>Broadway</family>
         <pointsize>13</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">border-color: rgb(0, 85, 0);
border-width: 1px;
border-style: solid;
color:rgb(0, 85, 0)</string>
       </property>
       <property name="text">
        <string>BROTO</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
       <property name="indent">
        <number>-14</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_3">
       <property name="font">
        <font>
         <family>Broadway</family>
         <pointsize>13</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">border-color: rgb(0, 85, 0);
border-width: 1px;
border-style: solid;
color: rgb(0, 85, 0);</string>
       </property>
       <property name="text">
        <string>TAM MÉDIO</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
       <property name="indent">
        <number>-14</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_4">
       <property name="font">
        <font>
         <family>Broadway</family>
         <pointsize>13</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">border-color: rgb(0, 85, 0);
border-width: 1px;
border-style: solid;
color: rgb(0, 85, 0);</string>
       </property>
       <property name="text">
        <string>TAM. GRANDE</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
       <property name="indent">
        <number>-14</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_5">
       <property name="font">
        <font>
         <family>Broadway</family>
         <pointsize>13</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">border-color: rgb(0, 85, 0);
border-width: 1px;
border-style: solid;
color:rgb(0, 85, 0)</string>
       </property>
       <property name="text">
        <string>SUPERPLANTA</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
       <property name="indent">
        <number>-14</number>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="superplanta">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>370</y>
      <width>451</width>
      <height>91</height>
     </rect>
    </property>
    <property name="sizeIncrement">
     <size>
      <width>0</width>
      <height>0</height>
     </size>
    </property>
    <property name="baseSize">
     <size>
      <width>0</width>
      <height>0</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>SimSun-ExtB</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="tabletTracking">
     <bool>true</bool>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="ButtonNovaplanta">
    <property name="geometry">
     <rect>
      <x>639</x>
      <y>393</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>SIM!</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>151</width>
      <height>131</height>
     </rect>
    </property>
    <property name="sizeIncrement">
     <size>
      <width>5</width>
      <height>0</height>
     </size>
    </property>
    <property name="font">
     <font>
      <pointsize>6</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 0, 0);</string>
    </property>
    <property name="text">
     <string>⠀⠀⠀⣿⣿⡀
⣤⣤⣤⣿⣿⣿⣦⣤⣾⣦⣤⡀⠀⠀⠀⠀⠀⢀⣤⣤⣄
⠛⠻⣿⣿⣿⣿⣿⣿⣿⣿⣿⣧⡀⠀⠀⠀⢀⡜⠛⠛⠛
⠀⠀⠀⠈⠻⣿⣿⣿⣿⡿⠋⢹⣿⡆⠀⢀⣼⣿⣿⣿⣷⣶⣶⣶
⠀⠀⠀⠀⠀⠈⠉⠉⠉⠀⠀⢸⣿⣿⣶⣿⣿⣿⣿⣿⣿⣿⠟⠁
⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠸⢿⣿⡏⠉⠙⠿⠿⠿⠟⠁
⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⢈⣹⡇
⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠈⣹⣇⡀
⠀⠀⠀⠀⠀⢀⣀⣀⣀⣀⣀⣠⣾⣿⣿⣷⣶⣶⣶⣦⡀
⠀⠀⠀⢀⣴⣿⣿⣿⣿⣿⣿⣿⣿⣿⣿⣿⣿⣿⣿⣿⣿
⠀⠀⠀⣿⣿⣿⣿⣿⣿⣿⣿⣿⣿⣿⣿⣿⣿⣿⣿⣿⣿
⠀⠀⠀⠙⠛⠛⠛⠛⠛⠛⠛⠛⠛⠛⠛⠛⠛⠛⠛⠛⠋</string>
    </property>
   </widget>
   <zorder>ButtonNovaplanta</zorder>
   <zorder>progressBar</zorder>
   <zorder>PLANTA</zorder>
   <zorder>pushButton</zorder>
   <zorder>label</zorder>
   <zorder>layoutWidget</zorder>
   <zorder>superplanta</zorder>
   <zorder>label_6</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
